### ISSUE TITLE
.es5 filetype is breaking with flowtype

### ISSUE THREAD

**Issue by madole** [joeybaker/react-element-query#12]
.es5 isn't a vaild filetype and breaks when running flow. 

```
import ElementQuery from 'react-element-query';
                              ^^^^^^^^^^^^^^^^^^^^^ react-element-query. Required module not found
```

Can we change it to be `.es5.js`?

**Comment by joeybaker** [comment]
I'm happy to take a PR converting it to just `.js`

**Comment by madole** [comment]
Awesome, Just testing it now 👍

**Comment by joeybaker** [comment]
Fixed in #13
